static/src/stylesheets/amp/_content.scss

```diff
@@ -0,0 +1,259 @@
+    box-sizing: border-box;
+}
+
+.content__labels {
+    box-sizing: border-box;
+    padding: $gs-baseline/2 0;
+    position: relative;
+    z-index: 1; // bring-to-front fix to make it clickable
+    overflow: hidden;
+}
+
+.content__labels--not-immersive {
+    border-bottom: 1px dotted $neutral-5;
+}
+
+.content__section-label {
+    float: left;
+    padding-right: $gs-gutter/3;
+
+    &,
+    .content__series-label {
+        float: left;
+    }
+}
+
+.content__series-label__link {
+    font-size: 1rem;
+    line-height: 1.25rem;
+    color: $neutral-2;
+}
+
+.content__headline {
+    font-size: 28px;
+    line-height: 32px;
+    font-weight: 400;
+    letter-spacing: -.02rem;
+    padding-top: $gs-baseline/6;
+    padding-bottom: $gs-baseline*2;
+
+    a {
+        &,
+        &:hover,
+        &:active,
+        &:focus {
+            color: $neutral-1;
+        }
+    }
+
+    em {
+        font-style: normal;
+    }
+
+    strong {
+        font-weight: normal;
+    }
+}
+
+.content__headline--byline {
+    display: block;
+    padding-top: 0;
+    margin-top: -$gs-baseline*2;
+    padding-bottom: $gs-baseline*2;
+
+    .byline-img {
+        float: right;
+        height: 9.375rem;
+        z-index: 2;
+        margin-left: -5rem;
+        margin-right: -2.5rem;
+    }
+}
+
+.content__standfirst {
+    font-size: 1rem;
+    line-height: 1.25rem;
+    margin-bottom: $gs-baseline/2;
+    color: $neutral-2;
+
+    > ul,
+    > ol {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+
+        > li {
+            position: relative;
+        }
+
+        * {
+            margin-bottom: $gs-baseline / 2;
+        }
+    }
+
+    > ol {
+        counter-reset: li;
+
+        >li {
+            text-indent: 18px;
+        }
+
+        > li:before {
+            position: absolute;
+            left: -16px;
+            content: counter(li);
+            counter-increment: li;
+            color: $neutral-3;
+        }
+    }
+}
+
+.content__dateline {
+    font-size: 12px;
+    line-height: 16px;
+    position: relative;
+    color: $neutral-2;
+    box-sizing: border-box;
+    padding-top: $gs-baseline / 6;
+    margin-bottom: $gs-baseline / 2;
+}
+
+.content__dateline-wpd--modified {
+    cursor: pointer;
+
+    &:hover,
+    &:active,
+    &:focus {
+        text-decoration: underline;
+    }
+}
+
+.content__meta-container {
+    min-height: gs-height(1);
+    position: relative;
+    margin-bottom: $gs-baseline;
+    border-top: 1px dotted $neutral-5;
+    border-bottom: 1px dotted $neutral-5;
+
+    .byline-img {
+        @include circular;
+        position: relative;
+        width: gs-span(1);
+        height: 60px; //Intentionally off grid
+        margin: $gs-baseline/2 0;
+        overflow: hidden;
+        background-color: $neutral-7;
+
+        // fixes for webkit not properly scaling/clipping the child element (still broken on some android browsers...)
+        -webkit-transform: translateZ(0);
```

Comment:
This is for legacy browser support, but potentially for AMP thats not needed. It was originally copied from here: 
https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/content/_content.scss#L455

I actually think this should be safe to delete.